### PR TITLE
Issue 604

### PR DESCRIPTION
**Pull request by zmcNotafraid**
1. 解析log 发现一个新的token后就会异步去获取他的name，symbol 等信息去导入到数据库
2. 访问udt api 的时候会异步获取total supply
3. 定时任务每一小时会去更新 updated_at < 一小时前的 udt 的信息
4. 桥接的代币会根据充值和提现记录每半小时去更新他们的total supply

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -70,11 +70,11 @@ config :godwoken_explorer, Oban,
     Oban.Plugins.Pruner,
     {Oban.Plugins.Cron,
      crontab: [
-       #  {"01 00 * * *", GodwokenIndexer.Worker.RefreshUDTSupply},
        {"10 00 * * *", GodwokenIndexer.Worker.DailyStat, args: %{datetime: nil}},
        {"*/2 * * * *", GodwokenIndexer.Worker.CheckLostAccount},
        {"*/10 * * * *", GodwokenIndexer.Worker.CheckContractCode},
-       {"* */4 * * *", GodwokenIndexer.Worker.RefreshNativeUDTSupply}
+       {"*/30 * * * *", GodwokenIndexer.Worker.RefreshBridgedUDTSupply},
+       {"* */1 * * *", GodwokenIndexer.Worker.UDTUpdater}
```
Comment (Keith-CY) [inline config/config.exs:77]: Please add the reason for updating `bridged udt total supply` and `udt total supply` with different intervals.

**Comment by Keith-CY** [comment]
Tests are missing

File: config/config.exs (hunk 1 above)
Comment (zmcNotafraid) [reply config/config.exs:77]: These two jobs have no connection. So the interval is free to set.

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
CI failed

**Review by Naupio — APPROVED** [review]

**Comment by Keith-CY** [comment]
Can we deploy this feature now? @zmcNotafraid

**Pull request merged**